libraries/helpers.rb

```diff
@@ -0,0 +1,38 @@
+module Chocolatey
+  module Helpers
+    # Get the ChocolateyInstall directory from the environment.
+    def chocolatey_install
+      ENV.fetch('ChocolateyInstall') { |env_var| machine_env_var(env_var) }
+    end
+
+    # The Chocolatey command.
+    #
+    # Reference: https://github.com/chocolatey/chocolatey-cookbook/pull/16#issuecomment-47975896
+    def chocolatey_executable
```

Comment:
Aren't you already breaking older chocolatey clients by using `chocolatey list`?

```ruby
  # choco /? returns an exit status of -1 with chocolatey 0.9.9 => use list
      cmd = Mixlib::ShellOut.new("#{chocolatey_executable} list -l chocolatey")
```